src/welcome.js

```diff
@@ -14,8 +15,15 @@ export class Welcome{
   }
 
   welcome(){
```

Comment:
@EisenbergEffect / @bryanrsmith -
What do you think about rename the `welcome` method to `submit`?  Reason I ask is when looking at the view model, it's hard to tell why/when "welcome" would get called and why welcome is squirreling away the current name value.  You have to look at the view to see that welcome is bound to the form's submit event.